apps/finance/app/src/components/AppLayout.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { AppBar, AppView, Button, ButtonIcon, Viewport, font } from '@aragon/ui'
+import MenuButton from './MenuButton/MenuButton'
+
+const AppLayout = ({
+  children,
+  title,
+  afterTitle,
+  smallViewPadding,
+  onMenuOpen,
+  mainButton,
+}) => {
+  return (
+    <Viewport>
+      {({ below }) => (
+        <AppView
+          padding={below('medium') ? smallViewPadding : 30}
```

Comment:
Maybe we could add a `largeViewPadding` that defaults to `30`?

Reply:
Yes good idea!